src/server/scripts/Northrend/IcecrownCitadel/boss_blood_prince_council.cpp

```diff
@@ -1498,6 +1498,27 @@ class spell_taldaram_ball_of_inferno_flame : public SpellScriptLoader
         {
             return new spell_taldaram_ball_of_inferno_flame_SpellScript();
         }
+
+        class spell_taldaram_ball_of_inferno_flame_AuraScript : public AuraScript
+        {
+            PrepareAuraScript(spell_taldaram_ball_of_inferno_flame_AuraScript);
+
+            void HandleStackDrop(AuraEffect const* /*aurEff*/, ProcEventInfo& /*eventInfo*/)
+            {
+                PreventDefaultAction();
+                GetTarget()->RemoveAuraFromStack(GetId());
```

Comment:
no need to, that function already does that and in case there is no stack left it removes the aura... so yeah no need to change it